mm/2s2h/BenGui/SearchableMenuItems.h

```diff
@@ -0,0 +1,1514 @@
+#include "2s2h/Enhancements/Enhancements.h"
+#include "2s2h/DeveloperTools/DeveloperTools.h"
+#include "2s2h/Enhancements/Graphics/3DItemDrops.h"
+#include "2s2h/BenGui/UIWidgets.hpp"
+#include "2s2h/BenGui/BenMenuBar.h"
+#include <variant>
+#include <tuple>
+
+extern "C" {
+#include "functions.h"
+extern PlayState* gPlayState;
+}
+
+typedef enum {
+    COLOR_WHITE,
+    COLOR_GRAY,
+    COLOR_DARK_GRAY,
+    COLOR_INDIGO,
+    COLOR_RED,
+    COLOR_DARK_RED,
+    COLOR_LIGHT_GREEN,
+    COLOR_GREEN,
+    COLOR_DARK_GREEN,
+    COLOR_YELLOW,
+} ColorOption;
+
+typedef enum {
+    DISABLE_FOR_CAMERAS_OFF,
+    DISABLE_FOR_DEBUG_CAM_ON,
+    DISABLE_FOR_DEBUG_CAM_OFF,
+    DISABLE_FOR_FREE_LOOK_ON,
+    DISABLE_FOR_FREE_LOOK_OFF,
+    DISABLE_FOR_AUTO_SAVE_OFF,
+    DISABLE_FOR_NULL_PLAY_STATE,
+    DISABLE_FOR_DEBUG_MODE_OFF,
+    DISABLE_FOR_NO_VSYNC,
+    DISABLE_FOR_NO_WINDOWED_FULLSCREEN,
+    DISABLE_FOR_NO_MULTI_VIEWPORT,
+    DISABLE_FOR_NOT_DIRECTX,
+    DISABLE_FOR_MATCH_REFRESH_RATE_ON,
+    DISABLE_FOR_MOTION_BLUR_MODE,
+    DISABLE_FOR_MOTION_BLUR_OFF,
+    DISABLE_FOR_FRAME_ADVANCE_OFF
+} DisableOption;
+
+struct widgetInfo;
+struct disabledInfo;
+using VoidFunc = void (*)();
+using DisableInfoFunc = bool (*)(disabledInfo&);
+using DisableVec = std::vector<DisableOption>;
+using WidgetFunc = void (*)(widgetInfo&);
+std::string disabledTempTooltip;
+const char* disabledTooltip;
+bool disabledValue = false;
+ColorOption menuThemeIndex = COLOR_INDIGO;
+const ImVec4 COLOR_NONE = { 0, 0, 0, 0 };
+
+typedef enum {
+    WIDGET_CHECKBOX,
+    WIDGET_COMBOBOX,
+    WIDGET_SLIDER_INT,
+    WIDGET_SLIDER_FLOAT,
+    WIDGET_CVAR_CHECKBOX,
+    WIDGET_CVAR_COMBOBOX,
+    WIDGET_CVAR_SLIDER_INT,
+    WIDGET_CVAR_SLIDER_FLOAT,
+    WIDGET_BUTTON,
+    WIDGET_COLOR_24, // color picker without alpha
+    WIDGET_COLOR_32, // color picker with alpha
+    WIDGET_SEARCH,
+    WIDGET_SEPARATOR,
+    WIDGET_SEPARATOR_TEXT,
+    WIDGET_WINDOW_BUTTON,
+    WIDGET_AUDIO_BACKEND, // needed because of special operations that can't be handled easily with the normal combobox
+                          // widget
+    WIDGET_VIDEO_BACKEND  // same as above
+} WidgetType;
+
+typedef enum {
+    MOTION_BLUR_DYNAMIC,
+    MOTION_BLUR_ALWAYS_OFF,
+    MOTION_BLUR_ALWAYS_ON,
+};
```

Comment:
Missing typedef name
```suggestion
} MotionBlurOption;
```